realm-annotations/src/main/java/io/realm/annotations/Index.java

```diff
@@ -17,8 +17,9 @@
 package io.realm.annotations;
 
 /**
- * The @Index annotation will add a search index to the field. A search index will make the
- * Realm file larger but search operations and queries will be faster.
+ * This annotation will add a search index to the field. A search index will make the
```

Comment:
Might be something here: http://gradle.1045684.n5.nabble.com/aggregating-javadocs-td1433469.html